samples/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '2'
+services:
+  zookeeper:
+    image: wurstmeister/zookeeper
```

Comment:
IMHO, [confluentinc/cp-zookeeper](https://hub.docker.com/r/confluentinc/cp-zookeeper) & [confluentinc/cp-kafka](https://hub.docker.com/r/confluentinc/cp-kafka) image seems to be more maintainable. and if the images are changed, the options will have to change too.

Reply:
Thanks for your opinion.